lib/Elevate/Components/ELS.pm

```diff
@@ -0,0 +1,53 @@
+package Elevate::Components::ELS;
+
+=encoding utf-8
+
+=head1 NAME
+
+Elevate::Components::ELS
+
+Remove ELS for CentOS 7
+
+=cut
+
+use cPstrict;
+
+use Elevate::OS ();
+
+use Cpanel::Pkgr  ();
+use Log::Log4perl qw(:easy);
+
+use parent qw{Elevate::Components::Base};
+
+use constant ELS_PACKAGE => 'els-define';
+
+sub pre_leapp ($self) {
+
+    return unless Elevate::OS::remove_els();
+
+    return unless Cpanel::Pkgr::is_installed(ELS_PACKAGE);
+
+    my @files_to_remove = qw{
+      /etc/yum/vars/elstoken
```

Comment:
@timmullin - I just got an updated version of the install script for ELS and with IP based licensing, this file is not needed.  Can you remove it from the list of files to remove please?

Reply:
I can go ahead and remove that.

Reply:
Now that I think about it, what if els had been installed with the older version of the script and the file is present?  Plus, we check if the file exists before deleting it; so, the file's absence would not create an error condition for the removal of els if they had used the newer version of the script.  It doesn't seem that there is any harm in leaving it in.  And, there are probably plenty of installs out there that have used the older version of the script and ought to have that file removed.